x/evm/migrations/v4/types/evm.pb.go

```diff
@@ -0,0 +1,4235 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: ethermint/evm/v1/evm.proto
```

Comment:
Execute `make proto-all` with migrated code

EDIT: I just saw that the whole `proto` folder is not migrated yet